cmd/res/profiles/llrp.device.profile.yaml

```diff
@@ -62,8 +62,8 @@ deviceResources:
       They contain the TagReportData or RFSurveyReportData,
       as well as any Custom parameter data.
     properties:
-      valueType: "Object" # not actually readable; it's async
-      readWrite: "R"
+      valueType: "Object" 
+      readWrite: "R" # not actually readable; it's async
```

Comment:
Comment needs to be moved to the `isHidden: true` once added and reworded to `resource is only async`. Same for others with this comment in both files.

Reply:
change has been made